src/Go/boardAnalysis/boardAnalysis.ts

```diff
@@ -548,29 +548,25 @@ export function findAdjacentLibertiesAndAlliesForPoint(
 }
 
 /**
- * Retrieves a simplified version of the board state. "X" represents black pieces, "O" white, and "." empty points.
+ * Retrieves a simplified version of the board state.
+ * "X" represents black pieces, "O" white, "." empty points, and "#" offline nodes.
  *
  * For example, a 5x5 board might look like this:
  * ```
- * [
- *   "XX.O.",
- *   "X..OO",
- *   ".XO..",
- *   "XXO..",
- *   ".XOO.",
- * ]
+ *   "XX.O.X..OO.XO..XXO...XOO."
```

Comment:
I think you changed the comments on the wrong place. This is still SimpleBoard, so still the stay form.